Restructure the task cleanup section (#1752)
theforeman/foreman-documentation: guides/common/modules/proc_configuring-the-cleaning-unused-tasks-feature.adoc

* Tasks that have run successfully and are older than thirty days
* All tasks that are older than a year

> You can configure the cleaning unused tasks feature using the below options:

Reviewer: ```suggestion
You can configure the cleaning unused tasks feature using these options:
```

* To configure the time at which {Project} runs the cron job, set the `--foreman-plugin-tasks-cron-line` parameter to the time you want in cron format.

> For example, to schedule the cron job to run every day at 15:00, enter the following command:

Reviewer: ```suggestion
For example, to schedule the cron job to run every day at 15:00, run the following command:
```

Author: @adamlazik1 I checked the docs and it seems that we use `enter` instead of `run` everywhere in the document. I checked the style guide and both seem valid. For the sake of consistency, should we keep this as `enter`?

Reviewer: Yes, sorry, this was my mistake.

+
[options="nowrap" subs="+quotes,attributes"]
----
# {foreman-installer} --foreman-plugin-tasks-cron-line "00 15 * * *"
----
* To configure the period after which {Project} deletes the tasks, edit the `:rules:` section in the `/etc/foreman/plugins/foreman-tasks.yaml` file.

> * To disable regular task cleanupon {Project}, enter the following command:

Reviewer 2: ```suggestion
* To disable regular task cleanupon {Project}, enter the following command:
+
```

this should fix the rendering of the code section

> * To disable regular task cleanupon {Project}, enter the following command:

Reviewer: ```suggestion
* To disable regular task cleanup on {Project}, enter the following command:
+
```

* Tasks that have run successfully and are older than thirty days
* All tasks that are older than a year

You can configure the cleaning unused tasks feature using these options:

* To configure the time at which {Project} runs the cron job, set the `--foreman-plugin-tasks-cron-line` parameter to the time you want in cron format.
For example, to schedule the cron job to run every day at 15:00, enter the following command:
+
[options="nowrap" subs="+quotes,attributes"]
----
# {foreman-installer} --foreman-plugin-tasks-cron-line "00 15 * * *"
----

> * To configure the period after which {Project} deletes the tasks, edit the `:rules:` section in the `/etc/foreman/plugins/foreman-tasks.yaml` file.

Reviewer: Should there be an example in code block for this?

Author: Hey @adamlazik1 I just copied the original text, but good idea! Let me check this on my instance and add an example.

Reviewer 2: I thought the same, but then also thought it'd spread the scope of this PR too wide?
(Not saying it shouldn't be done, just not here and today)

Reviewer: I am fine with either way, rest of the PR looks good to me. @AkshayGadhaveRH your choice.

Author: Hey @adamlazik1 @evgeni so I checked the option and tried to insert an example in the code, but it might need some tweaking so that it renders correctly. Let's add this at a later time?
